dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetPaymentMethodOnCartTest.php

```diff
@@ -60,6 +60,44 @@ public function testSetPaymentOnCartWithSimpleProduct()
         self::assertEquals($methodCode, $response['setPaymentMethodOnCart']['cart']['selected_payment_method']['code']);
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     * @magentoApiDataFixture Magento/GraphQl/Catalog/_files/simple_product.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/customer/create_empty_cart.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/add_simple_product.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/set_new_shipping_address.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/enable_offline_payment_methods.php
+     */
+    public function testSetPurchaseOrderPaymentMethodOnCartWithSimpleProduct()
```

Comment:
Please move tests to `\Magento\GraphQl\Quote\Customer\SetOfflinePaymentMethodsOnCartTest`
like it was in `\Magento\GraphQl\Quote\Customer\SetOfflineShippingMethodsOnCartTest`